infra/src/main/kotlin/fi/fta/geoviite/infra/linking/SwitchLinkingService.kt

```diff
@@ -1319,9 +1309,11 @@ private fun getSwitchBoundsFromSwitchLinkingParameters(
 
 data class LocationTrackChangesFromLinkingSwitch(
     val onlyDelinked: List<Pair<LocationTrack, LayoutAlignment>>,
-    val onlyTopoLinkEdited: List<LocationTrack>,
+    val onlyTopoLinkEdited: List<Pair<LocationTrack, LayoutAlignment>>,
     val alignmentLinkEdited: List<Pair<LocationTrack, LayoutAlignment>>,
-)
+) {
+    fun all() = onlyDelinked + onlyTopoLinkEdited + alignmentLinkEdited
```

Comment:
Tämän voisi nimetä uudestaan vaikka jollekin `allChanges()`-nimelle tms. Kotlinissa `all()` on myös `Boolean`:in palauttava listaoperaatio, niin piti lukea tuo rivin 1236 muutamaan kertaan ennen kuin tajusin miksi se ylipäätään kääntyy.